<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="EPUAKHAW0472" name="com.epam.preproduction.tests.CheckItemInformation" tests="1" failures="1" timestamp="11 Nov 2013 15:36:07 GMT" time="3,054" errors="0">
  <testcase name="testItemInformation" time="3,054" classname="com.epam.preproduction.tests.CheckItemInformation">
    <failure type="java.lang.NullPointerException" message="null value in entry: name=null">
      <![CDATA[java.lang.NullPointerException: null value in entry: name=null
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:235)
	at com.google.common.collect.ImmutableMap.entryOf(ImmutableMap.java:144)
	at com.google.common.collect.ImmutableMap.of(ImmutableMap.java:89)
	at org.openqa.selenium.remote.RemoteWebElement.getAttribute(RemoteWebElement.java:122)
	at com.epam.preproduction.helpers.CheckItemInformationTestHelper.gerUrls(CheckItemInformationTestHelper.java:60)
	at com.epam.preproduction.helpers.CheckItemInformationTestHelper.verifyItemLinksAreEqual(CheckItemInformationTestHelper.java:110)
	at com.epam.preproduction.tests.CheckItemInformation.testItemInformation(CheckItemInformation.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:842)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1166)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1178)
	at org.testng.TestRunner.privateRun(TestRunner.java:757)
	at org.testng.TestRunner.run(TestRunner.java:608)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1158)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1083)
	at org.testng.TestNG.run(TestNG.java:999)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:74)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:92)
	at org.apache.maven.surefire.Surefire.run(Surefire.java:180)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireBooter.runSuitesInProcess(SurefireBooter.java:350)
	at org.apache.maven.surefire.booter.SurefireBooter.main(SurefireBooter.java:1021)
]]>
    </failure>
  </testcase>
</testsuite>
